.docker/source/Dockerfile

```diff
@@ -20,3 +20,6 @@ RUN mkdir src && \
     rosdep update && \
     rosdep install -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO} --as-root=apt:false && \
     rm -rf /var/lib/apt/lists/*
+    #
+    # Build the workspace
+    colcon build --symlink-install --event-handlers console_direct+
```

Comment:
This is a new command, no? Line 22 is missing a `\`.
Though I think building the workspace would be worth having as a separate layer (RUN).

Reply:
Good catch. Fixed